types/coin.go

```diff
@@ -176,9 +184,18 @@ func (coins Coins) Minus(coinsB Coins) Coins {
 	return coins.Plus(coinsB.Negative())
 }
 
-// IsGTE returns True iff coins is NonNegative(), and for every
-// currency in coinsB, the currency is present at an equal or greater
-// amount in coinsB
+// IsGTE returns True iff for every denom in coins, the denom is present at a
```

Comment:
```suggestion
// IsGT returns True iff for every denom in coins, the denom is present at a
```

Reply:
I also don't quite understand these APIs.

```
assert.True(t, Coins{{"A", one}, {"B", one}}.IsGT(Coins{{"B", one}}))
```

`A` is not present in the "other" coins.  Shouldn't this return false?

Reply:
IMHO no, the function's name should be `AllGT` to mean that the comparison is performed on all denoms.

Reply:
So the godoc should be updated.